packages/frontend/src/components/wallet-migration/Disable2FA.jsx

```diff
@@ -160,6 +189,8 @@ const Disable2FAModal = ({ handleSetActiveView, onClose }) => {
                 }
             </Container>
         </Modal>
+        { failedAccount && <AccountLockModal accountId={failedAccount} onClose={onAccountLockClose} onComplete={onAccountLockComplete} /> }
```

Comment:
<img width="416" alt="image" src="https://user-images.githubusercontent.com/25015977/192958740-1fc6c19c-cd92-496a-9cbf-d8352df0070d.png">

It would be nice if there was a loading button on top of the continue button. In this case, I entered an invalid key phrase. There should be an error message telling the user that we were unable to unbrick the account due to a wrong key provided.

Reply:
Don't worry about displaying this error. I am creating a ticket for it and will handle it as soon as you push this out. There are other places where error handling needs to be done properly so will take care of it all in one.